test/spec/functional/chaiPromises.js

```diff
@@ -2,16 +2,14 @@ describe('chai-as-promised', function() {
 
     /**
      *  Remove the `should` global on Object.prototype to allow chai.should for these tests,
-     *  and set up chai, chai-as-promised and chai.should properly
+     *  and set up chai and chai.should properly
      */
 
     require('should').noConflict();
 
     var chai = require('chai');
-    var chaiAsPromised = require('chai-as-promised');
 
     chai.Should();
-    chai.use(chaiAsPromised);
```

Comment:
Because my tests weren't passing without it.  :P  Maybe just because I was
trying to run one test in isolation?
On Sep 11, 2015 18:30, "Christian Bromann" <notifications@github.com> wrote:

> In test/spec/functional/chaiPromises.js
> <https://github.com/webdriverio/webdriverio/pull/779#discussion_r39324905>
> :
>
> >
> >      chai.Should();
> > -    chai.use(chaiAsPromised);
>
> Why did you changed that? Have to revert it. No worries just fyi [image:
> :wink:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webdriverio/webdriverio/pull/779/files#r39324905>.
>